integration-test/src/test/java/org/sagebionetworks/IT049FileHandleTest.java

```diff
@@ -550,6 +558,34 @@ public void testMultipartUploadV2Reset() throws FileNotFoundException, SynapseEx
 		assertFalse(startStatus.getUploadId().equals(statusAgain.getUploadId()));
 	}
 
+	// See PLFM-5769

```

Comment:
For this test, we create an "external" S3 location inside the default bucket, and we try to upload a file to this location. This test captures the failure case in PLFM-5769.